A model of the phenological development of winter oilseed rape (Brassica napus L.)
==================================================================================
Habekott√© 1997
--------------
Developed an empirical model to predict Brassica napus phenology.

Used just temperature data for the periods:
  + sowing to emergence
  + first open flower to end of flowering
  + end of flowering to maturity

Used temperature, vernalization and photoperiod to explain the period emergence to first open flower.


Delayed flowering time in Arabidopsis and Brassica rapa by the overexpression of FLOWERING LOCUS C (FLC) homologs isolated from Chinese cabbage (Brassica rapa L. ssp. pekinensis)
==================================================================================================================================================================================
Kim et al. 2007
---------------
Isolated three Brassica rapa FLC genes from Chiifu:
  + BrFLC1 (BrFLC.A10)
  + BrFLC2 (BrFLC.A2)
  + BrFLC3 (BrFLC.A3a)

MADS-domain showed higher homology to Arabidopsis thaliana FLC gene than regions outside the MADS-domain.

BrFLC.A10 and BrFLC.A2 were found to be ubiquitously expressed in leaves, shoot tips and roots. BrFLC.A3a was more lowly expressed than BrFLC.A10 and BrFLC.A2 in leaves and shoot tips and was not expressed in roots.

Compared an early flowering, an intermediate flowering and a late flowering Chinese cabbage. BrFLC.A10 and BrFLC.A2 were more highly expressed than BrFLC.A3a in all lines. Expression of all three FLC genes was lower in the early flowering variety compared to the other two varieties before vernalization. Vernalization for 40 days caused a decrease in all three BrFLC genes in all varieties.

In transgenic experiments using Arabidopsis thaliana and Brassica rapa, all three BrFLC genes cause a delay in flowering, with BrFLC.A2 having the strongest effect.


Tuning growth cycles of Brassica crops via natural antisense transcripts of BrFLC
=================================================================================
Li et al. 2015
--------------
Found antisense transcripts of Brassica rapa FLC2.

Overexpression of the transcripts in Brassica rapa caused loss of vernalization and a conversion of biennial crop to annual.

Leaf number, branch number and seed yield were also affected by the transcript overexpression.


Quantitative trait loci for flowering time and morphological traits in multiple populations of Brassica rapa
============================================================================================================
Lou et al. 2007
---------------
Performed QTL analysis with three different populations:
  + F2/3 - Cross between rapid cycling line and a vegetable type Chinese cabbage line
  + DH-38 - Cross between oil type Yellow sarson line and a vegetable type Pak choi line
  + DH-30 - Cross between oil type Yellow sarson line and a vegetable type turnip line

Found eight flowering time QTLs identified using different populations. Of note:
  + *populations QTL was identified in; candidate gene co-localized with QTL*
  + F2/3, DH-38, DH-30; Brassica rapa FLC2
  + DH-30; LFY homologue
  + DH-38; Brassica rapa FLC1
  + DH-30; VRN2 homologue
  + F2/3; FT homologue

Strong correlations between different turnip traits and flowering time

Flowering time QTL were sensitive to environment. The QTL identified near Brassica rapa FLC2, which explained the largest percentage of the observed variation in some experiments, explained less of the variance after vernalization.
  
Co-localization of QTL for seed traits, growth-related traits, leaf traits, flowering time and turnip formation implies pleiotropy.


The Brassica rapa FLC homologue FLC2 is a key regulator of flowering time, identified through transcriptional co-expression networks
====================================================================================================================================
Xiao et al. 2013
----------------
Found that flowering time genes are preferentially retained in the Brassica rapa genome.

Used a population which was a cross between oil type Yellow sarson line and a vegetable type Pak choi line.

QTL near Brassica rapa FLC2 explains 33.5% of the observed variation in flowering time.

At the FLC2 (A2) locus, several eQTL were found for:
  + FT (Two copies on A7; trans; floral integrator)
  + SOC1 (A3 and A5; trans; floral integrator)
  + CO (A10; trans; cirdadean clock)
  + SRR1 (A2; cis; cirdadean clock)
  + TCP11 (A2; cis; cirdadean clock)

Found that the genes listed above as well as PI (A2; floral meristem gene) and PGI (A1; cirdadean clock) had expression levels which correlated with the flowering time phenotype.

Found eight eQTL hotspots associated with the expression of flowering time related genes.


Comparative Analysis of FLC Homologues in Brassicaceae Provides Insight into Their Role in the Evolution of Oilseed Rape
========================================================================================================================
Zou et al. 2012
---------------
Identified nine FLC homologues in Brassica napus which fall into clades corresponding to the whole-genome triplication events:
  + BnFLC.A10 (BrFLC1) (Clade I)
  + BnFLC.A2 (BrFLC2) (Clade II)
  + BnFLC.A3a (BrFLC3) (Clade III)
  + BnFLC.A3b (BrFLC5) (Clade III)
  + BnFLC.C9a (BoFLC1) (Clade I)
  + BnFLC.C9b (BoFLC1) (Clade I)
  + BnFLC.C3a (BoFLC3) (Clade III)
  + BnFLC.C2 (BoFLC4) (Clade II)
  + BnFLC.C3b (BoFLC5) (Clade III)

The promoter regions of the BnFLC genes were more divergent to each other than exonic sequence.

A 75-bp segment crucial for expression of FLC in nonvernalized Arabidopsis thaliana plants was absent from BnFLC.C9a.

Transcripts for BnFLC.C2 and BnFLC.C3a could only be isolated from Westar.

BnFLC.C3b might be a pseudogene as a transcript could not be isolated from either Tapidor, Ningyou7 or Westar.

Looked at expression of BnFLC.A2, BnFLC.A3b, BnFLC.A10 and BnFLC.A3a-BnFLC.C3a:
  + All showed a decrease in expression after vernalization in Tapidor
  + Expression patterns of different copies differed over time in Ningyou7
  + BnFLC.A3a-BnFLC.C3a showed the greatest fold change between vernalized and nonvernalized measurements
  + BnFLC.A3b showed a significant expression change between Tapidor and Ningyou7, attributed to differential splicing

Found that BnFLC.A3b, BnFRI.A3 and a Brassica napus homologue of CBF1 clustered at the site of a flowering time QTL.

Lou et al. 2012
---------------

Circadean clock genes are preferentially retained relative to comparison groups of their neighbouring genes, a set of randomly chosen genes and a set of housekeeping genes

gene dosage hypothesis

schiessl et al 2015
-------------------

looked at flowering, yield and plant height and identified a number of flowering time genes

also found enrichment for RNA interference and processing GO terms of genes in regions associated with flowering time

for yield componants, 30% of the candidate genes were detected with more than one copy, caused by several regions of interest being homoeologous indicating co-expression of those genes between the sub-genomes


osborn et al 2003 - review
-----------------

how do polyploids get an immediate selective advantage
variation introducted due to:
* increased variation in dosage-regulated gene expression
* altered regulatory mechanisms
* rapid genetic and epigenetic changes
variation in flowering time may result in the polyploid species being reproductively isolated

pires et al 2004
----------------

resynthesised brassica napus and found an early and a late flowering population within 6 generations

no evidence that BnFLC1 (BrFLC A10 and BoFLC C9) contributed to the differences in flowering time between the B. napus lines.

BnFLC2 (BrFLC2 A2, BoFLC2 C2) - BoFLC C2 seems to be missing. The early-flowering polyploid population has a chromosome translocation so has two copies of A2, whereas the Late-flowering population also have a chromosome translocation and have lost A2. Hypothesise that the gene has diverged to another function or is interfering with other FLC copies

BnFLC3 (A3, C3) early flowering lines have lost BoFLC3 and have a double dose of BrFLC3 - accounted for a significant proportion of the observed variation

BoFLC5 (A3 and C3) was silenced. This was expected, as the gene was silenced in the diploid plants, but the rapa copy was not silenced - no effect on the divergence in flowering time

higgins et al 2012
------------------

RNA-Seq to look at A and C genome transcription

there were differentially expressed homoeologues, with enriched GO terms not associated with gene dosage effects

dun et al 2014
--------------

novel function related to male fertility probably arose
through altered protein function rather than through
changes in gene expression patterns.

fertility-restoring effect on the
B. napus male-sterile mutant 7365A

the C9 copy in oleracea and napus gained a novel role in male sterility while also maintaining the ancestral role, while the other copies just maintained the ancestral role


Axelsson et al. 2001
--------------------

qtl analysis of flowering time found association with 2 of 3 replicated segments containing both FLC and CO in each diploid genome


Schiessl et al 2014
-------------------

sequence variation, copy number variation, presence absence variation

used sequence variation between Brassica napus varieties with different vernalization requirements to suggest gene copies which are less functional

most flowering time gene
copies arising from the ancestor genomes were retained after
allopolyploidization, and many of the retained paralogs are still
expressed

Liu et al. 2014
---------------

Greater TEs accumulation in B. oleracea than B. rapa

WGT-produced paralogous genes are over-retained in GO categories associated with regulation of metabolic and biosynthetic processes, RNA metabolism and transcription factors

phylogenetic analyes have shown that the whole genome triplication has led to an expansion of genes involved in flowering time control
